### PR TITLE
Add info how to use KubeVirt inv plugin

### PR DESCRIPTION
**Pull request by machacekondra**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by machacekondra** [comment]
@pkliczewski @mmazur

File: README.md [hunk 1]
```diff
@@ -76,3 +76,37 @@ $ ansible-playbook tests/playbooks/k8s_facts_vm.yml
...
+
+```
+plugin: kubevirt
+connections:
+  - namespaces:
+      - default
+    network_name: default
+```
+
+This configuration will use kubevirt plugin and will list all VMi from default namespaces and as default IP it will use network interface from default network.
```
Comment (pkliczewski) [inline README.md]: I would change here to: `plugin and will list all VMIs from default namespace and...`

File: README.md [hunk 2]
```diff
@@ -76,3 +76,37 @@ $ ansible-playbook tests/playbooks/k8s_facts_vm.yml
...
+
+### Execute the playbook with KubeVirt inventory plugin
+To use the plugin in playbook, just pass it to `ansible-playbook` command in `-i` option:
+
+```
+$ ansible-playbook -i kubevirt.yaml myplaybook.yml
+```
+
+Note that KubeVirt inventory plugin is designed to work with multus, for VMis connective from the same network as you run the playbook. For VMis exposed by Kubernetes service, please use [k8s](https://docs.ansible.com/ansible/latest/plugins/inventory/k8s.html).
```
Comment (pkliczewski) [inline README.md]: I would change here to: `for VMIs reachable from the same network ...` and `For VMIs exposed by Kubernetes services, please...`

**Comment by machacekondra** [comment]
Fixed your comments.

**Pull request merged**